packages/core/src/pie-chart.ts

```diff
@@ -232,6 +208,30 @@ export class PieChart extends BaseChart {
 					.style("opacity", 1);
 			});
 
+		// fade out left callout
+		this.innerWrap.select("g.callout-lines-left")
+			.transition()
+			.duration(Configuration.transitions.default.duration / 2)
```

Comment:
is this a new duration? Should be added to `Configurations` like so:
![image](https://user-images.githubusercontent.com/14989804/62318048-f1552b80-b468-11e9-9cf8-64807e19a63a.png)

and this function would get the transition:
![image](https://user-images.githubusercontent.com/14989804/62318074-fb772a00-b468-11e9-85f9-4383643bff2c.png)